<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>837</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scro">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>329</y>
      <width>501</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>499</width>
       <height>449</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>724</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ys_btn">
       <property name="text">
        <string>영상보기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_btn">
       <property name="text">
        <string>삭제</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="detail_image_lab1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>391</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>581</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Clothes</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QCheckBox" name="clo_Poncho">
       <property name="text">
        <string>Poncho</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="clo_parka">
       <property name="text">
        <string>Parka</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="clo_cardigan">
       <property name="text">
        <string>Cardigan</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="clo_tank">
       <property name="text">
        <string>Tank</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="clo_jersey">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Jersey</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="clo_joggers">
       <property name="text">
        <string>Joggers</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="clo_hoodie">
       <property name="text">
        <string>Hoodie</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="clo_blazer">
       <property name="text">
        <string>Blazer</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="clo_top">
       <property name="text">
        <string>Top</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="clo_jacket">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Jacket</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="clo_tee">
       <property name="text">
        <string>Tee</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="clo_sweater">
       <property name="text">
        <string>Sweater</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="clo_all">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>All</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="clo_turtleneck">
       <property name="text">
        <string>Turtleneck</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="clo_blouse">
       <property name="text">
        <string>Blouse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="clo_down">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>581</width>
      <height>52</height>
     </rect>
    </property>
    <property name="title">
     <string>Color</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="color_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Violet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pink</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="color_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Violet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pink</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="color_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Violet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pink</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>581</width>
      <height>52</height>
     </rect>
    </property>
    <property name="title">
     <string>Time</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QDateTimeEdit" name="time_to">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2018</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="maximumDateTime">
        <datetime>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
         <year>2100</year>
         <month>12</month>
         <day>31</day>
        </datetime>
       </property>
       <property name="minimumDateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1900</year>
         <month>9</month>
         <day>14</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd h:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="time_from">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2018</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="maximumDate">
        <date>
         <year>2100</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>1900</year>
         <month>9</month>
         <day>14</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd h:mm:ss</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="verticalGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="nh_btn">
       <property name="text">
        <string>녹화</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ssg_btn">
       <property name="text">
        <string>실시간 영상 보기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="jjd_btn">
       <property name="text">
        <string>저장된 영상 보기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="log_btn">
       <property name="text">
        <string>로그 보기</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="detail_view">
    <property name="geometry">
     <rect>
      <x>557</x>
      <y>273</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>상세보기 창</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>30</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="search_list">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>273</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>탐색결과 리스트 (미리보기 창)</string>
    </property>
   </widget>
   <widget class="QLabel" name="detail_videotime">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>335</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>영상 촬영시간 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="detail_mantime">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>420</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>인물 등장시간 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="detail_clo">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>510</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>등장인물 의상정보 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="detail_color">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>600</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>등장인물 색상정보 :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
